pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```

Comment:
hmm weird may need to take a look what's the difference between 6.1 and 6.0

Reply:
In my local when I run `pnpm i` it will auto change to `6.1`

Reply:
which pnpm version are you using?

Reply:
`8.6.0`

Reply:
please upgrade to the latest